subsys/bluetooth/common/Kconfig

```diff
@@ -100,10 +100,11 @@ config BT_BUF_ACL_RX_COUNT
 
 config BT_BUF_EVT_RX_SIZE
 	int "Maximum supported HCI Event buffer length"
-	default $(UINT8_MAX) if (BT_EXT_ADV && BT_OBSERVER) || BT_PER_ADV_SYNC || BT_DF_CONNECTION_CTE_RX || BT_CLASSIC
+	default $(UINT8_MAX) if (BT_EXT_ADV && BT_OBSERVER) || BT_PER_ADV_SYNC || BT_DF_CONNECTION_CTE_RX || BT_CLASSIC || BT_CHANNEL_SOUNDING
 	# LE Read Supported Commands command complete event.
 	default 68
 	range 68 $(UINT8_MAX)
+	range 78 $(UINT8_MAX) if BT_CHANNEL_SOUNDING
```

Comment:
Just to make sure it's not a typo: You mentioned 76 in https://github.com/zephyrproject-rtos/zephyr/pull/78764/files#r1771154273 but sets it to 78 here. Is that correct?

Reply:
76 bytes for parameters (as mentioned) + 2 bytes for the header ?